content/blog/aws-cdk-on-pulumi-1.0/index.md

```diff
@@ -0,0 +1,273 @@
+
+For example, the following program deploys an AWS Apprunner CDK Construct using Pulumi:
+
+```typescript
+import * as pulumi from '@pulumi/pulumi';
+import * as pulumicdk from '@pulumi/cdk';
+import { Service, Source } from '@aws-cdk/aws-apprunner-alpha';
+
+class AppRunnerStack extends pulumicdk.Stack {
+    url: pulumi.Output<string>;
+
+    constructor(id: string, options?: pulumicdk.StackOptions) {
+        super(id, options);
+
+        const service = new Service(this, 'service', {
+            source: Source.fromEcrPublic({
+                imageConfiguration: { port: 8000 },
+                imageIdentifier: 'public.ecr.aws/aws-containers/hello-app-runner:latest',
+            }),
+        });
+
+        this.url = this.asOutput(service.serviceUrl);
+    }
+}
+
+const app = new pulumicdk.App('app', (scope: pulumicdk.App): pulumicdk.AppOutputs => {
+    const stack = new AppRunnerStack('teststack');
+    return {
+        url: stack.url,
+    };
+});
+export const url = app.outputs['url'];
+```
+
+Deploy with a `pulumi up`:
+
+```console
+> pulumi up
+
+Updating (dev)
+
+View Live: https://app.pulumi.com/lukehoban/pulumi-cdk-apprunner/dev/updates/1
+
+     Type                                   Name                       Status
+ +   pulumi:pulumi:Stack                    pulumi-cdk-apprunner-dev   created
+ +   └─ cdk:index:StackComponent            teststack                  created
+ +      └─ cdk:construct:Service            teststack/adapter/service  created
+ +         └─ aws-native:apprunner:Service  service6D174F83            created
+
+Outputs:
+    url: "2ez3iazupm.us-west-2.awsapprunner.com"
+
+Resources:
+    + 4 created
+```
+
+And curl the endpoint:
+
+```console
+> curl https://$(pulumi stack output url)
+
+   ______                             __        __      __  _                  __
+  / ____/___  ____  ____ __________ _/ /___  __/ /___ _/ /_(_)___  ____  _____/ /
+ / /   / __ \/ __ \/ __ `/ ___/ __ `/ __/ / / / / __ `/ __/ / __ \/ __ \/ ___/ /
+/ /___/ /_/ / / / / /_/ / /  / /_/ / /_/ /_/ / / /_/ / /_/ / /_/ / / / (__  )_/
+\____/\____/_/ /_/\__, /_/   \__,_/\__/\__,_/_/\__,_/\__/_/\____/_/ /_/____(_)
+                 /____/
+
+        Congratulations, your service has successfully deployed on AWS App Runner.
+
+Open it in your browser at https://2ez3iazupm.us-west-2.awsapprunner.com/
+
+Try the workshop at https://apprunnerworkshop.com
+Read the docs at https://docs.aws.amazon.com/apprunner
+```
+
+## Pulumi takes CDK apps to the next level
+
+Combine CDK constructs with any of the features of Pulumi programs to deploy faster, easier and to any cloud. Use Pulumi functions and stack references to connect to pre-existing infrastructure and mix in resources from any provider to bring *all* of your infrastructure under management.
+
+For example, we can use CDK's `ecs_patterns` to quickly create a loadbalanced Fargate service, but route it with a record in an exisiting CloudFlare DNS zone.
```

Comment:
`route it` reads a little weird to me, is `routing` a verb that can be done to a service? Perhaps `route to it` or `route traffic to it`

Reply:
```suggestion
For example, we can use CDK's `ecs_patterns` to quickly create a loadbalanced Fargate service, but route traffic to it with a record in an exisiting CloudFlare DNS zone.
```